### PR TITLE
CASSANDRA-7814: Added indexes to table and ks metadata 

### PR DESCRIPTION
**Pull request by stef1927**
This is the suggested change for https://datastax-oss.atlassian.net/browse/PYTHON-241.

### CODE REVIEW

File: cassandra/metadata.py [hunk 1]
```diff
@@ -215,6 +216,20 @@ def _build_usertype(self, keyspace, usertype_row):
         return UserType(usertype_row['keyspace_name'], usertype_row['type_name'],
                         usertype_row['field_names'], type_classes)
 
+    def _add_table_metadata_to_ks(self, keyspace_metadata, row, col_rows, trigger_rows):
```
Comment (aholmberg) [inline cassandra/metadata.py]: Wondering if this functionality would be better implemented in ``KeyspaceMetadata``, where the index is held (KeyspaceMetadata.add_table(self, table_meta), or something like that).

Comment (stef1927) [inline cassandra/metadata.py]: Yes you are quite right, I moved both methods to the ks metadata, see latest commit.

**Comment by aholmberg** [comment]
Please see updates in #284. Let me know what you think of those changes.

**Comment by stef1927** [comment]
Changes look good and tests for CASSANDRA-7814 still pass, thanks!

**Pull request merged**